cclib/parser/logfileparser.py

```diff
@@ -398,16 +401,12 @@ def updateprogress(self, inputfile, msg, xupdate=0.05):
                 self.progress.update(newstep, msg)
                 self.progress.step = newstep
 
+    @abstractmethod
     def normalisesym(self, symlabel):
-        """Standardise the symmetry labels between parsers.
+        """Standardise the symmetry labels between parsers."""
 
-        This method should be overwritten by individual parsers, and should
-        contain appropriate doctests. If is not overwritten, this is detected
-        as an error by unit tests.
-        """
-        raise NotImplementedError("normalisesym(self, symlabel) must be overriden by the parser.")
-
-    def float(self, number):
+    @staticmethod
+    def float(number):
```

Comment:
If this is not parser-specific then maybe move out of the class? Into `utils`?

Reply:
It's not parser-specific, but it's going to change ~80 lines in the parsers from `self.float(...)` to `utils.float(...)`. Are you ok with that?

Reply:
I am - I think we do a lot of that for other functions. But maybe in a separate PR, then.